lib/challenge.js

```diff
@@ -1,5 +1,35 @@
 'use strict';
 
+var stringNum;
+var reversedString;
+var reversedNumber;
+var numberInBinary;
+var sum;
+
+let isPalindrome = function isPalindrome(number) {
+  stringNum = number.toString();
+  reversedString = stringNum.split('').reverse().join('');
+  reversedNumber = Number(reversedString);
+  return number === reversedNumber;
+};
+
+let convertToBinary = function convertToBinary(number) {
+  numberInBinary = number.toString(2);
+  return numberInBinary;
+};
+
+let bothPalindromic = function bothPalindromic() {
+  for (let i = 0; i < 1000000; i++) {
+    if (isPalindrome(i) && isPalindrome(convertToBinary(i))) {
+      sum += i;
+    }
+  }
+};
+
+bothPalindromic();
+console.log(sum);
+
```

Comment:
Another good practice would be to try and separate our data from our logic; we could do this by storing `1000000` inside a constant, or even by passing in an arbitrary `max` value.